pkg/scheduler/eventhandlers.go

```diff
@@ -270,8 +270,10 @@ func addAllEventHandlers(
 				case *v1.Pod:
 					return assignedPod(t)
 				case cache.DeletedFinalStateUnknown:
-					if pod, ok := t.Obj.(*v1.Pod); ok {
-						return assignedPod(pod)
+					if _, ok := t.Obj.(*v1.Pod); ok {
+						// The carried object may be stale, so we don't use it to check if
+						// it's assigned or not. Attempting to cleanup anyways.
+						return true
```

Comment:
just to make sure, so when both filters (this and the one below) return true, both handlers will be invoked, right?

Reply:
correct. The handlers are independent of each other.